gitpod-io/website
# Improve a11y on navigation

## Description
Adds various HTML semantics + ARIA roles so that screen readers can better comprehend our navigation. More of these PRs are to follow, because to be fully up-to-spec will take a lot more effort, although shipping this skateboard will help quite a bit 🤞.

## Related Issue(s)


This is a part of a11y efforts like https://github.com/gitpod-io/website/pull/2632, https://github.com/gitpod-io/website/pull/2408 or https://github.com/gitpod-io/website/pull/2302.


## How to test
Go over the navigation with an assistive technology like [VoiceOver](https://support.apple.com/en-asia/guide/voiceover/welcome/mac) and see how usable it is if you couldn't use a mouse.

## Further reading

- A good guide for accessible menus: https://www.smashingmagazine.com/2017/11/building-accessible-menu-systems/

## Release Notes

```release-note
NONE
```


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2635"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2635"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
